src/components/CVcomponents/HeaderCV.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import EditIcon from "../../images/edit_icon_orange.png"
+
+class HeaderCV extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
I have a feeling you are mixing state and props here. I am not sure if this is incorrect, but I am personally used to writing them separately.